regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -112,6 +113,7 @@ var CommentReviewView = Backbone.View.extend({
         }).get();
         toggleInput($text, $valid.length === 0);
         toggleInput($select, $valid.length > 0);
+        toggleInput($selectDropdown, $valid.length > 0);
```

Comment:
after reading this stuff some more I realize how wrong (though weirdly functional) this was. The end goal should be something that just removes the `display: none;`  from the div with the `select-dropdown` class when the `<select id="gov_agency_select"` is enabled.